Calibration/HcalCalibAlgos/src/HcalIsoTrkAnalyzer.cc

```diff
@@ -62,41 +65,48 @@
 
 //#define DebugLog
 
-class HcalIsoTrkAnalyzer : public edm::EDAnalyzer {
+class HcalIsoTrkAnalyzer : public edm::one::EDAnalyzer<edm::one::SharedResources> {
```

Comment:
You will find that (begin/end)(Run/LuminosityBlock) methods will not be called. See https://twiki.cern.ch/twiki/bin/view/CMSPublic/FWMultithreadedFrameworkOneModuleInterface for details on how to see those transitions.